action.yml

```diff
@@ -11,35 +11,55 @@ runs:
           source ${{ github.action_path }}/flag-alt-text.sh
 
           if [ ${{ github.event.comment }} ]; then
-            comment=$COMMENT
-            issue=${{ github.event.issue.html_url }}
-            comment_owner=${{ github.event.comment.user.login }}
-            is_pr=${{ github.event.issue.pull_request.url != '' }}
+            content=$COMMENT
+            issue_url=${{ github.event.issue.html_url }}
+            user=${{ github.event.comment.user.login }}
+            if ${{ github.event.issue.pull_request.url != '' }}; then
+              type=pr_comment
+            else
+              type=issue_comment
+            fi
             target=${{ github.event.comment.html_url }}
           elif [ ${{ github.event.issue && !github.event.comment }} ]; then
-            comment=$ISSUE_BODY
-            issue=${{ github.event.issue.html_url }}
-            comment_owner=${{ github.event.issue.user.login }}
-            is_pr=false
+            content=$ISSUE_BODY
+            issue_url=${{ github.event.issue.html_url }}
+            user=${{ github.event.issue.user.login }}
+            type=issue_description
             target=" your issue body"
           elif [ ${{ github.event.pull_request && !github.event.comment }} ]; then
-            comment=$PR_BODY
-            issue=${{ github.event.pull_request.html_url }}
-            comment_owner=${{ github.event.pull_request.user.login }}
-            is_pr=true
+            content=$PR_BODY
+            issue_url=${{ github.event.pull_request.html_url }}
+            user=${{ github.event.pull_request.user.login }}
+            type=pr_description
             target=" your pull request body"
+           elif [ ${{ github.event.discussion && !github.event.comment }} ]; then
```

Comment:
My only thought is since this is the only option using graphql what if we just moved the graphql command into this if statement?